src/app/header/header.component.scss

```diff
@@ -2,10 +2,9 @@
 @import 'src/assets/styles/font';
 
 .header-container {
-  padding: 0 24px;
   border: 1px solid $black-12;
   width: 1555.2px;
-  height: 68px;
+  height: 68.8px;
```

Comment:
please try to avoid decimal numbers here.

Reply:
Resolved.............Thank you